README.adoc

```diff
@@ -28,6 +28,40 @@ docker run -p 8080:8080 --rm -it -e SPRING_PROFILES_ACTIVE=dev rostering
 ----
 This takes a few minutes on the first time. The application will be automatically populated with test data.
 
+## Development
```

Comment:
Formatting should follow adoc, not GitHub:
https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74